diff --git a/xmake.lua b/xmake.lua
index aecf2ce7..6f70c975 100644
--- a/xmake.lua
+++ b/xmake.lua
@@ -144,9 +144,8 @@ elseif is_plat("macosx") then
     end
 end
 
-add_requires("myboost " .. boost_version, {
+add_requires("boost", {
   system = false,
-  alias = "boost",
   debug = is_mode("debug"),
   configs = {
     shared = is_plat("windows") and true or false,
@@ -158,9 +157,7 @@ add_requires("myboost " .. boost_version, {
     pyver = get_config("pyver"),
   },
 })
-if is_plat("windows") then
-  add_requireconfs("myboost.python", {override = true, system=false})
-end
+add_requireconfs("boost.python", {override = true, system=false})
 
 add_requires("spdlog", {system = false, configs = {header_only = true, fmt_external = true, vs_runtime = "MD"}})
 add_requireconfs("spdlog.fmt", {override = true, version = fmt_version, configs = {header_only = true}})
